pkg/cli/node.go

```diff
@@ -431,12 +435,11 @@ func runDecommissionNodeImpl(
 		MaxBackoff:     20 * time.Second,
 	}
 
-	// Marking a node as fully decommissioned is driven by a two-step process.
-	// We start off by marking each node as 'decommissioning'. In doing so,
-	// replicas are slowly moved off of these nodes. It's only after when we're
-	// made aware that the replica counts have all hit zero, and that all nodes
-	// have been successfully marked as 'decommissioning', that we then go and
-	// mark each node as 'decommissioned'.
+	// Marking a node as fully decommissioned is driven by a three-step process.
+	// 1) Mark each node as 'decommissioning'. In doing so, replicas are slowly moved off of these nodes.
+	// 2) After when we're made aware that the replica counts have all hit zero, and that all nodes
+	// have been successfully marked as 'decommissioning', drain each node.
+	// 3) After each node has been drained, mark each node as 'decommissioned'.
```

Comment:
Maybe a few words here about if the node we're talking to is the one being decommissioned, and why that's frowned upon+skips the drain step? Also [nit] wrap these comments at 80 chars.

Reply:
Done.